background.js

```diff
@@ -88,6 +91,13 @@ if(initStorage("v0.8.92", true)) {
     // v0.8.92
 	initStorage("popupDelay", "150");
 	initStorage("showOnKey", "");
+
+
+	// v???
+	initStorage("ttsEnabled", "false");
+	if (localStorage['ttsEnabled'] == "yes") {

```

Comment:
Just fyi, that you don't need this `== "yes"` part for new options. It was only used when I changed from using "yes" to "true".

Reply:
(I fixed it already though)